ns-2.35/apps/swifi.cc

```diff
@@ -84,8 +84,70 @@ SWiFiAgent::SWiFiAgent() : Agent(PT_SWiFi), seq_(0), mac_(0)
 	target_ = 0;
 	bind("packet_size_", &size_);
 	bind("slot_interval_", &slot_interval_);
+}	
+
+   
+//Schedule Max Weight policy
+SWiFiAgent::Schedule_Max_Weight()	  
```

Comment:
Just a small note: rename the function name to `scheduleMaxWeight` for more consistency.